src/Sema.zig

```diff
@@ -23236,7 +23247,7 @@ fn validateExternType(
         .Float,
         .AnyFrame,
         => return true,
-        .Pointer => return !ty.isSlice(),
+        .Pointer => return if (ty.isSlice()) false else sema.validateExternType(ty.childType(), position),
```

Comment:
This check is not correct, only slices and pointers to comptime only types are not extern compatible.

Reply:
@Vexu are there any suitable abi for type '*u3'?

Reply:
The ABI type of `*u3` is just a regular pointer.